mosviz/viewers/mos_viewer.py

```diff
@@ -312,9 +313,49 @@ def load_selection(self, row):
             A `row` object representing a row in the MOS catalog. Each key
             should be a column name.
         """
-        spec1d_data = nirspec_spectrum1d_reader(row['spectrum1d'])
-        spec2d_data = nirspec_spectrum2d_reader(row['spectrum2d'])
-        image_data = acs_cutout_image_reader(row['cutout'])
+
+        if "loaders" in self.catalog.meta:
+            # if loader is specified
+            if "spec1d" in self.catalog.meta["loaders"]:
+                # check .glue/config.py for custom loaders
+                if hasattr(config, self.catalog.meta["loaders"]["spec1d"]):
```

Comment:
Note that if you do it this way, you don't need to import the user's `config.py` file, and should be using Glue's separate config module (e.g. `from glue import config`). Just in case I wasn't clear!